src/raspberry_pi_driver/behaviors.py

```diff
@@ -2,8 +2,14 @@
 from time import sleep
 
 from src.raspberry_pi_alarm.buzzer_interface import Buzzer
+from src.raspberry_pi_camera.camera_interface import CameraInterface
 from src.raspberry_pi_motion_sensor.motion_interface import MotionPir
 
+livestream_on = False
+recording_on = False
+# Create camera object
+cam = CameraInterface()
```

Comment:
Suggestion: consider put `livestream_on` and `recording_on` in a NamedTuple in main function of `rpi_out_driver.py`. You can call it `camera_flags`.

Consider also instantiate `cam` in the same main function. Then, you can pass `camera_flags` and `cam` as arguments to your `record` and `livestream` behaviors.

This way, you can still modify the states of your flags, use `cam`, but we avoid global variables and make `behaviors.py` cleaner.

Reply:
Done.